component/s3storage/client.go

```diff
@@ -45,10 +46,12 @@ import (
 	"github.com/Seagate/cloudfuse/internal/stats_manager"
 
 	"github.com/aws/aws-sdk-go-v2/aws"
+	awsHttp "github.com/aws/aws-sdk-go-v2/aws/transport/http"
```

Comment:
What does awsHTTP mean on this line?

Reply:
Yeah, the awsHttp represents the name of the library. It would normally be named http, but net/http is also imported so their would be a name collision. So this just renames the one for aws.